### DISCUSSION TITLE
using a common service in multiple jobs

### DISCUSSION THREAD

**Discussion by bonesoul** [community/community#28683] [Actions]
Basically U do run a postgresql service within my workflow and run migrations against it to get a ready to serve database.

What I'd like to know is if it's possible to use the same postgresql instance within multiple jobs in my workflow file.

**Comment by airtower-luna** [comment]
That's not possible, a service container is discarded at the end of the job.

Depending on what you need the database for you could either add further steps to the same job that does the migrations, or export the database as an artifact and import it in further jobs (basically like you'd do a backup/restore).
